grpc-server/grpc_server.go

```diff
@@ -17,52 +14,29 @@ import (
 	"google.golang.org/grpc/reflection"
 )
 
-// GetCerts returns a TLS certificate, PEM bytes, and file modification time for a
-// given path. An error is returned for any failure.
+// GetCerts returns a TLS certificate.
+// An error is returned for any failure.
```

Comment:
Nit: could oneline this now that it is shorter
```suggestion
// GetCerts returns a TLS certificate. An error is returned for any failure.
```